android/libraries/rib-base/src/test/java/com/badoo/ribs/core/NodeTest.kt

```diff
@@ -82,6 +78,14 @@ class NodeTest {
         node.children.addAll(allChildren)
     }
 
+
+    private fun attachToViewAlongWithChildren() {
```

Comment:
I checked this tests, and looks like you don't need to connect 3 childViews.
Consider in future replace it with tests with only 2 childs, from my point of view it will simplify support and you will have to read less code.

Reply:
I do need three for the back press handling tests unfortunately.